service/src/main/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseService.java

```diff
@@ -692,6 +693,24 @@ public Map<String, Object> removeCourtBundleByBundleId(CaseData caseData,
         }
     }
 
+    public Map<String, Object> removeCorrespondenceDocument(CaseData caseData,
+                                                            String migrationId,
+                                                            UUID expectedDocumentId) {
+
+        List<Element<SupportingEvidenceBundle>> newCorrespondenceDocuments =
+            caseData.getCorrespondenceDocuments().stream()
```

Comment:
not sure if relevant, but just crossed my mind - is this document in any of the other collections, do we have a non-confidential correspondence collection?